### PR TITLE
Modernize a bit

### PR DESCRIPTION
**Pull request by Goooler**
Kill buildSrc & enable configuration cache to speed up.

### CODE REVIEW

File: gradle/DependencyUpdates.gradle.kts [hunk 1]
```diff
@@ -31,15 +33,15 @@ enum class ReleaseType(private val level: Int) {
 
 object DependencyUpdates {
     private val stableKeywords = arrayOf("RELEASE", "FINAL", "GA")
-    private val releaseRegex = "^[0-9,.v-]+(-r)?$".toRegex(RegexOption.IGNORE_CASE)
+    private val releaseRegex = "^[\\d,.v-]+(-r)?$".toRegex(RegexOption.IGNORE_CASE)
     private val rcRegex = releaseKeywordRegex("rc")
     private val betaRegex = releaseKeywordRegex("beta")
     private val alphaRegex = releaseKeywordRegex("alpha")
     private val devRegex = releaseKeywordRegex("dev")
```
Comment (Goooler) [inline gradle/DependencyUpdates.gradle.kts:40]: Maybe we can remove `com.github.ben-manes.versions` plugin, use @renovate-bot instead.

**Comment by chrisbanes** [comment]
Thanks for doing this!

**Review by chrisbanes — APPROVED** [review]

File: gradle/DependencyUpdates.gradle.kts (hunk 1 above)
Comment (chrisbanes) [reply gradle/DependencyUpdates.gradle.kts]: I've looked at Renovate before but didn't have the time to set it up 👍

Comment (Goooler) [reply gradle/DependencyUpdates.gradle.kts]: Just enable it and remove all `dependencyUpdates` usages, after that we can kill the `buildSrc`.

**Pull request merged**